scenarios/rag/custom-rag-app/README.md

```diff
@@ -0,0 +1,146 @@
+---
+page_type: sample
+languages:
+- python
+products:
+- ai-model-inference
+- ai-search
+- ai-evaluation
+description: Custom rag app sample using Azure AI SDKs
+---
+
+## Custom RAG App
+
+### Overview
+
+This folder contains a reference implementation of a custom RAG app built using the Azure Foundry AI SDK.
+
+Check out the following files for implementation details:
+ - [create_search_index.py](): creates an Azure AI search index 
+    - Uses data in [assets/products.csv]() to generate the index
+ - [get_product_documents.py](): implements document retreival based on user queries
+    - Using [assets/intent_mapping.prompty]() for generating a search query from the conversation
+ - [chat_with_products.py](): implements chat that grounds response in retrieved documents
+    - Uses [assets/grounded_chat.prompty]() for formatting retrieved documents into the chat prompt
+    - Implements the [Microsoft AI Chat protocol](https://github.com/microsoft/ai-chat-protocol) for compatibility with evaluation and front-ends
+
```

Comment:
Could be worth adding in a link to relevant documentation near the start of this sample, similar to what's in the jupyter template.

![image](https://github.com/user-attachments/assets/655a6a3b-4fa8-4bfc-9509-62903a94f878)